### ISSUE TITLE
¿Existe un NuGet?

### ISSUE THREAD

**Issue by 15mgm15** [thiagomoises/moisesToolkit.MercadoPago.NetCore#1]
Buenas!

Quisiera probar tu librería, de casualidad tienes un NuGet?

**Comment by alefcarlos** [comment]
Yes: https://www.nuget.org/packages/Moises.Toolkit.MercadoPago.NetCore/2.0.0-preview0011

**Comment by thiagomoises** [comment]
Sorry for delay in the response. I've been busy for the past few months.
I will update the readme with docs and references
